### DISCUSSION TITLE
FileUpload Dont Work With SVG Image

### DISCUSSION THREAD

**Discussion by OmarASharaf** [filamentphp/filament#2435] [Help]
### Package

filament/filament

### Package Version

v2.12.9

### Laravel Version

v9

### Livewire Version

v2.6

### PHP Version

PHP 8

### Bug description

when I upload an image of type SVG its stops on uploading and don't complete upload the image

note: I am using the simple resource

![Capture](https://user-images.githubusercontent.com/18222965/168444992-a802b110-9e76-4483-b804-028cffeb5bba.JPG)

**Comment by danharrin** [comment]
any console errors?

**Comment by OmarASharaf** [comment]
there is no error on the console.
I tried uploading all the types of images and they worked fine, but the SVG upload is not working

**Comment by danharrin** [comment]
I've just tried uploading an SVG and it works fine?

**Comment by dinandmentink** [comment]
Came here with the same issue. Since a recent release (probably 2.17.49 but I'm not sure) svg uploads do not work anymore. There's no console error and no network request. 

All other images work. I do use s3 hosting (no cors issues, since all other images work). 

Edit: this is an older issue. Did the original author manage to fix this? I have been using svgs for months now and suddenly am unable to. May this be regression? Just trying to find a solution.

Edited before unsubscribe:

In my case there was an actual issue with the svg itself. Notably, svgs from [storyset](https://storyset.com/) seem to have this issue. I don't know why, and probably there's something off in filepond. But I no longer use storyset so I'm fine.

**Comment by lknightnet** [comment]
are there any solutions to this problem?

**Comment by avinmaster** [comment]
+1
